src/app/error/global-error-handler.service.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { TestBed } from '@angular/core/testing';
+import { GlobalErrorHandlerService } from './global-error-handler.service';
+import { HttpErrorResponse } from '@angular/common/http';
+
+describe('GlobalErrorHandlerService', () => {
+
+  let globalErrorHandlerService: GlobalErrorHandlerService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [GlobalErrorHandlerService]
+    });
+
+    globalErrorHandlerService = TestBed.get(GlobalErrorHandlerService);
+  });
+
+  it('should be created', () => {
+    const service: GlobalErrorHandlerService = TestBed.get(GlobalErrorHandlerService);
+    expect(service).toBeTruthy();
+  });
+
+  it('should take message text from ErrorEvent.message in case of client-side error', () => {
+    const message = 'expected text';
+    const errorEvent = new ErrorEvent('', {
+      message: message
+    });
+    const err: HttpErrorResponse = new HttpErrorResponse({
+      error: errorEvent
+    });
+    expect(globalErrorHandlerService.createFullMessage(err)).toBe(message);
+  });
+
+  it('should take message text from HttpErrorResponse.message in case of an error from other servers', () => {
```

Comment:
Is `HttpErrorResponse.message`a misprint? Should `HttpErrorResponse.statusText` be here?

Reply:
thank you. done